docs/repositories.md

```diff
@@ -23,7 +23,12 @@ Some ways you can use this:
 
 - Run the Bazel-managed version of node: `bazel run @nodejs_host//:node path/to/program.js`
 - Run the Bazel-managed version of npm: `bazel run @nodejs_host//:npm`
-- Run the Bazel-managed version of yarn: `bazel run @nodejs_host//:yarn`
+
+## @yarn
+
+This repository is created by calling the `yarn_repositories` function in your `WORKSPACE` file.
+
+- Run the Bazel-managed version of yarn: `bazel run @yarn//:yarn`
```

Comment:
@mattem here's what you were looking for

Reply:
Ah nice 👍